engine/src/main/scala/cromwell/jobstore/JobStoreReader.scala

```diff
@@ -1,28 +1,36 @@
 package cromwell.jobstore
 
-import akka.actor.{Actor, ActorLogging}
+import akka.actor.{Actor, ActorLogging, Props}
 import akka.event.LoggingReceive
-import cromwell.core.JobKey
+import cromwell.backend.BackendJobDescriptorKey
+
+object JobStoreReader {
+  def props() = Props(new JobStoreReader())
+}
 
 class JobStoreReader extends Actor with ActorLogging {
   override def receive = LoggingReceive {
-    case QueryJobCompletion(jobKey) => sender ! JobNotComplete(jobKey)
+    case QueryJobCompletion(jobKey) =>
+      sender ! JobNotComplete(jobKey)
+      // TODO PBE Decide if we want to make this a singleton that stays alive or a do-and-die actor
```

Comment:
IMO (and I suppose also TOL) we're at a point where we should stop making things "TODO PBE" and just do them and/or make them as tickets. In this case I'd suggest tech talking it for interested parties.